spec/support/db_housekeeping.rb

```diff
@@ -18,6 +18,7 @@ def self.clean(seed: true)
       cases_users_transitions_trackers
       linked_cases
       search_queries
+      report_types
```

Comment:
Notice how report_types is now being cleared, because the user is expected to use `create_report_type` rather than the factory method directly (`create :report_type`)